source/new.rst

```diff
@@ -211,8 +211,8 @@ Alice and Bob.
 
 2. Bob receives the bootstrap code and
 
-   a) If Bob's device knows a key that matches ``Alice_FP``
-      the protocol continues with 4b)
+   a) If Bob's device already verified that ``Alice_FP``
```

Comment:
I think the term verified is confusing here.
The main use case is a situation
where Bob already received Autocrypt headers from Alice
and now they verify (eventually for the first time).

```suggestion
   a) If Bob's device already knows a key with the fingerprint ``Alice_FP``
      that
```